From 7ffc1eba8596544d6e3af0fd5f4004fff6d56f84 Mon Sep 17 00:00:00 2001
From: Naveen Saini <naveen.kumar.saini@intel.com>
Date: Wed, 29 Nov 2023 15:48:57 +0800
Subject: [PATCH] hwloc_detection.cmake: remove cross-compiation check

This check is preventing pkgconfig to
search for hwloc under cross-compilation.

pkgconfig able to find hwloc pkg from sysroot.

Upstream-Status: Inappropriate

Signed-off-by: Naveen Saini <naveen.kumar.saini@intel.com>
---
 cmake/hwloc_detection.cmake | 2 --
 1 file changed, 2 deletions(-)

diff --git a/cmake/hwloc_detection.cmake b/cmake/hwloc_detection.cmake
index 47233b17..5a7e84c3 100644
--- a/cmake/hwloc_detection.cmake
+++ b/cmake/hwloc_detection.cmake
@@ -46,8 +46,6 @@ endforeach()
 unset(HWLOC_TARGET_NAME)
 
 if (NOT HWLOC_TARGET_EXPLICITLY_DEFINED AND
-    # No hwloc auto detection for cross compilation
-    NOT CMAKE_CROSSCOMPILING AND
     NOT TBB_DISABLE_HWLOC_AUTOMATIC_SEARCH
 )
     find_package(PkgConfig QUIET)
-- 
2.37.3

